config/routes.rb

```diff
@@ -1,3 +1,10 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+ 
+get "xml", :to => "static#xml"
```

Comment:
@jtellis1991 It sounds like maybe you resolved the issue with this route being the right format, but you can force the route to default to `xml` format instead of HTML

```suggestion
get "xml", :to => "static#xml", :defaults => { :format => 'xml' }
```

then you shouldn't need to specify the format in the controller.